src/applications/personalization/dashboard/containers/DashboardApp.jsx

```diff
@@ -196,6 +196,7 @@ class DashboardApp extends React.Component {
     if (this.props.profile.verified) {
       this.props.getEnrollmentStatus();
     }
+    document.getElementById('dashboard-title').focus();
```

Comment:
not sure if there's a benefit to do this with React refs, but doing it with vanilla JS works.